New page: aria-orientation (#10797)
mdn/content: files/en-us/web/accessibility/aria/attributes/aria-orientation/index.md

---
title: 'aria-orientation'
slug: Web/Accessibility/ARIA/Attributes/aria-orientation
tags: 
  - Accessibility
  - ARIA
  - ARIA attribute
  - ARIA property
  - aria-orientation
  - Reference
---

The `aria-orientation` attribute indicates whether the element's orientation is horizontal, vertical, or unknown/ambiguous.

## Description

> For some widgets, it is important for the user to know the orientation to know how to navigate the widget as orientation impacts the expected behaviors of the left, right, up and down arrows. The `aria-orientation` attribute is used to indicate to assistive technology users whether an element's orientation is `horizontal` or `vertical`, or `undefined`. 

Reviewer: ```suggestion
It may be important for the user to know the orientation to know how to navigate certain widgets, as orientation impacts the expected behaviors of the left, right, up and down arrows. The `aria-orientation` attribute is used to indicate to assistive technology users whether an element's orientation is `horizontal` or `vertical`, or `undefined`. 
```
